client/tests/integration/upgrade.rs

```diff
@@ -144,9 +144,9 @@ fn executor_upgrade_should_revoke_removed_permissions() -> Result<()> {
 
     // Check that permission exists
     assert!(client
-        .request(FindPermissionSchema)?
+        .request(FindExecutorDataModel)?
         .permissions()
-        .contains(&can_unregister_domain_token.definition_id));
+        .contains(&can_unregister_domain_token.id));
```

Comment:
```suggestion
        .contains(&can_unregister_domain_token.id()));
```

you should use method access in client